Core/Source/MVVM/Model.swift

```diff
@@ -18,494 +18,41 @@ public protocol Model {
     var modelVersion: ModelVersion { get }
 }
 
-/// 128-bit value that represents the version of the data in a model.
+/// Swift 4.2+ Hasher-computed value that represents the version of the data in a model.
```

Comment:
Adding a comment here in hopes that Github will not hide loading this in the diff by default as it's the largest part of the change.